aws-distributed-configuration/build.gradle.kts

```diff
@@ -4,6 +4,6 @@ plugins {
 
 dependencies {
     api(projects.awsCommon)
-    api(mn.micronaut.discovery)
+    api(libs.micronaut.discovery.client)
```

Comment:
Once we have a snapshot of https://github.com/micronaut-projects/micronaut-discovery-client/pull/407 we can change this to use the BOM

Reply:
Ok, we have a snapshot...  updated... lets watch CI